include/dav1d/headers.rs

```diff
@@ -543,38 +543,69 @@ pub(crate) const RAV1D_TRC_UNKNOWN: Rav1dTransferCharacteristics = DAV1D_TRC_UNK
-pub(crate) const _RAV1D_MC_BT2020_NCL: Rav1dMatrixCoefficients = DAV1D_MC_BT2020_NCL;
-pub(crate) const _RAV1D_MC_SMPTE_YCGCO: Rav1dMatrixCoefficients = DAV1D_MC_SMPTE_YCGCO;
-pub(crate) const _RAV1D_MC_SMPTE240: Rav1dMatrixCoefficients = DAV1D_MC_SMPTE240;
-pub(crate) const _RAV1D_MC_BT601: Rav1dMatrixCoefficients = DAV1D_MC_BT601;
-pub(crate) const _RAV1D_MC_BT470BG: Rav1dMatrixCoefficients = DAV1D_MC_BT470BG;
-pub(crate) const _RAV1D_MC_FCC: Rav1dMatrixCoefficients = DAV1D_MC_FCC;
-pub(crate) const RAV1D_MC_UNKNOWN: Rav1dMatrixCoefficients = DAV1D_MC_UNKNOWN;
-pub(crate) const _RAV1D_MC_BT709: Rav1dMatrixCoefficients = DAV1D_MC_BT709;
-pub(crate) const RAV1D_MC_IDENTITY: Rav1dMatrixCoefficients = DAV1D_MC_IDENTITY;
+pub const DAV1D_MC_IDENTITY: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Identity as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_BT709: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Bt709 as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_UNKNOWN: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Unknown as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_FCC: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Fcc as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_BT470BG: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Bt470bg as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_BT601: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Bt601 as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_SMPTE240: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Smpte240 as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_SMPTE_YCGCO: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Smpte_ycgco as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_BT2020_NCL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Bt2020_ncl as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_BT2020_CL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Bt2020_cl as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_SMPTE2085: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Smpte2085 as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_CHROMAT_NCL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Chromat_ncl as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_CHROMAT_CL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Chromat_cl as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_ICTCP: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Ictcp as Dav1dMatrixCoefficients;
+pub const DAV1D_MC_RESERVED: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::Reserved as Dav1dMatrixCoefficients;
+
+#[derive(Clone, Copy, PartialEq, Eq, FromRepr)]
+pub enum Rav1dMatrixCoefficients {
+    Identity = 0,
+    Bt709 = 1,
+    Unknown = 2,
+    Fcc = 4,
+    Bt470bg = 5,
+    Bt601 = 6,
+    Smpte240 = 7,
+    Smpte_ycgco = 8,
+    Bt2020_ncl = 9,
+    Bt2020_cl = 10,
+    Smpte2085 = 11,
+    Chromat_ncl = 12,
+    Chromat_cl = 13,
+    Ictcp = 14,
+    Reserved = 255,
```

Comment:
this `Reserved` variant is not in the spec (https://aomediacodec.github.io/av1-spec/av1-spec.pdf, page 118/119), is this a C implementation detail maybe?

Reply:
Yeah, that does seem weird, especially since 3 is supposed to be reserved.